_posts/2016-04-14-welcoming-jscs-to-eslint.md

```diff
@@ -1,4 +1,5 @@
 ---
+layout: post
```

Comment:
A question - why was this added?

Reply:
I saw newer posts had that and thought I needed it for the date to show. However I found that wasn't needed. I can remove that if you would like.

Reply:
You're right, newer posts have that.

It seems that in https://github.com/eslint/website/pull/649 we removed that field from all posts, but we didn't update the [template](https://github.com/eslint/eslint/blob/e3cd1414489ceda460d593ac7e7b14f8ad45d4fc/templates/blogpost.md.ejs#L2) for release blog posts, so new release posts keep producing it.

Reply:
I think it's overridden at the directory level [here](https://github.com/eslint/website/blob/d44ac60c666fd1dc07ec1350bed667544af902ae/_posts/_posts.json#L2), so it might be unused at the end. Not sure how that works, though.

Either way, I can't see a difference between posts that have that field and posts that don't have it. If you also don't see a difference, then it might be better to remove it, so that we don't get confused later about why was it added in this PR.

Reply:
Removed that line from the post.